polygen.opam

```
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+name: "polygen"
+version: "1.0.6"
+synopsis: "Polygen"
+description: "Polygen — a first effort towards satyre in computer science."
+maintainer: "Paolo Donadeo <p.donadeo@gmail.com>"
+authors: [ "Alvise Spanò <spano.alvise@gmail.com>" ]
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+dev-repo: "git+https://github.com/alvisespano/Polygen.git"
+post-messages: [
+  "Polygen installed in %{bin}%/polygen"
+  {success & !user-setup:installed}
+]
+depends: [
+  "js_of_ocaml"     { >= "3.9.0"}
+  "js_of_ocaml-ppx" { >= "3.9.0"}
+  "dune"            { build & >= "2.8.2"}
```

```suggestion
  "dune"            {>= "2.8.2"}
```
Dune does not support the `{build}` tag. See https://github.com/ocaml/dune/issues/2147

---

As always thanks @kit-ty-kate